covasim/webapp/index.html

```diff
@@ -330,9 +337,9 @@ <h4 class="my-2">Campaign Builder</h4>
           </svg>
           <span class="sr-only">View this project on GitHub</span>
         </b-nav-item>
-        <b-nav-item v-b-modal.licenses>License</b-nav-item>
-        <b-nav-item :href="app.org_url + '/terms'" target="_blank">Terms</b-nav-item>
-        <b-nav-item :href="app.org_url + '/privacy'" target="_blank">Privacy</b-nav-item>
+        <b-nav-item v-b-modal.licenses><span class="nav-text">License</span></b-nav-item>
+        <b-nav-item :href="app.org_url + '/terms'" target="_blank"><span class="nav-text">Terms</span></b-nav-item>
+        <b-nav-item :href="app.org_url + '/privacy'" target="_blank"><span class="nav-text">Privacy</span></b-nav-item>
       </b-navbar-nav>
```

Comment:
Overriding the nav-text class and adding the spans is unnecessary. Instead, just override the nav-link class.

Reply:
Agree, added to https://github.com/InstituteforDiseaseModeling/covasim/issues/180